src/ClientData/DataManagerTest.cpp

```diff
@@ -28,7 +28,7 @@ static void CallMethodOnDifferentThreadAndExpectDeath(DataManager& data_manager,
 
 TEST(DataManager, CanOnlyBeUsedFromTheMainThread) {
   DataManager data_manager;
-  FunctionInfo function{"path/to/module", "buildid", 12, 16, "foo()"};
+  FunctionInfo function{"path/to/module", "buildid", 12, 16, "foo()", false};
```

Comment:
(nit) Here and in the tests below:

```suggestion
  FunctionInfo function{"path/to/module", "buildid", /*address=*/12, /*size=*/16, "foo()", /*is_hotpatchable=*/false};
```